### PR TITLE
Redirects for old urls

### PR DESCRIPTION
**Pull request by mitzimorris**
Created a file called `url_redirects.txt` which follows the format of https://github.com/stan-dev/docs/blob/master/redirects.txt, containing a mapping of URLS on the current mc-stan.org website to the new version of the website.

As part of this effort, harvested a few more paragraphs of content from pages on the old website and added them to the `about/index.qmd` page.    I also reordered the content on the about page - I put the licensing information first, inasmuch as this is something that matters to users who matter.  Of course, all users matter, so open to other ordering of the topics.

### CODE REVIEW

**Comment by WardBrian** [comment]
We could also use the [Quarto redirects](https://quarto.org/docs/websites/website-navigation.html#redirects) feature for any of these where the target on the new site is not a `#` anchor. 

If we prefer consistency and just putting them all here, that's fine by me. 

Note (to me as much as anyone): Currently, the [script](https://github.com/stan-dev/docs/blob/master/generate_redirects.py) used in the docs doesn't like blank lines. We can either separate the sections here with a empty comment line `# ` or we can update the script when pulling it out into a shared repo between these two.

**Comment by WardBrian** [comment]
(the current script also hardcodes that this file is called `redirects.txt`, so it will probably want an update either way)

**Comment by mitzimorris** [comment]
using quarto redirects whenever possible, file `redirects.txt` lists those links which go to some internal page section.

File: learn-stan/stancon-talks.qmd [hunk 1]
```diff
@@ -2,6 +2,28 @@
 title: StanCon Talks
 page-layout: full
 toc-location: right
+
+aliases:
+ - ../events/stancon2019Cambridge/index.html
+ - ../events/stancon2019Cambridge/abstracts.html
```
Comment (WardBrian) [inline learn-stan/stancon-talks.qmd]: The links for a specific year could link to the header, e.g. both of these could be in redirects.txt with ` https://mc-stan.org/learn-stan/stancon-talks.html#stancon-2019-cambridge`

File: tools/index.qmd [hunk 2]
```diff
@@ -2,6 +2,41 @@
...
 page-layout: full
 toc-location: right
+
+aliases:
+ - ../interfaces/loo.html 
+ - ../interfaces/rstanarm.html 
+ - ../interfaces/pystan.html 
+ - ../interfaces/mathematica-stan.html 
+ - ../interfaces/rstantools.html 
+ - ../interfaces/math.html
```
Comment (WardBrian) [inline tools/index.qmd]: Similarly, if we really wanted to, this could link to `/tools/#developer-tools-and-apis`

**Review by WardBrian — COMMENTED** [review]
A couple minor comments about better places to redirect to.

It's hard to check something like this for completeness -- Here's every html file in the current site (downloaded from https://github.com/stan-dev/stan-dev.github.io/deployments/github-pages and then ran through `tree`):

```
./404.html
./about/funding/index.html
./about/index.html
./about/logo/index.html
./about/sgb/minutes.html
./about/team/index.html
./citations/index.html
./community/index.html
./developers/index.html
./development/index.html
./documentation/case-studies/curse-dims.html
./documentation/case-studies/dina_independent.html
./documentation/case-studies/divergences_and_bias.html
./documentation/case-studies/dorazio-royle-occupancy.html
./documentation/case-studies/hierarchical_2pl.html
./documentation/case-studies/identifying_mixture_models.html
./documentation/case-studies/Latent_class_case_study.html
./documentation/case-studies/mbjoseph-CARStan.html
./documentation/case-studies/mle-params.html
./documentation/case-studies/pcm_and_gpcm.html
./documentation/case-studies/pool-binary-trials.html
./documentation/case-studies/pool-binary-trials-rstanarm.html
./documentation/case-studies/radon.html
./documentation/case-studies/rasch_and_2pl.html
./documentation/case-studies/rsm_and_grsm.html
./documentation/case-studies/sem.html
./documentation/case-studies/soil-knit.html
./documentation/case-studies/tutorial_twopl.html
./documentation/case-studies/weakly_informative_shapes.html
./documentation/index.html
./events/index.html
./events/stancon2017-notebooks/stancon2017-ahn-haines-zhang-hBayesDM.html
./events/stancon2017-notebooks/stancon2017-groves-irt.html
./events/stancon2017-notebooks/stancon2017-lei-sanders-dawson-ad-attribution.html
./events/stancon2017-notebooks/stancon2017-margossian-gillespie-ode.html
./events/stancon2017-notebooks/stancon2017-nicenboim-vasishth-retrieval-models.html
./events/stancon2017-notebooks/stancon2017-sanders-lei-mass-shootings.html
./events/stancon2017-presenter-bios.html
./events/stancon2018/index.html
./events/stancon2018/report.html
./events/stancon2018/stancon-code_of_conduct.html
./events/stancon2018/stancon-submissions.html
./events/stancon2018Helsinki/GpuOptimizedMath.html
./events/stancon2018Helsinki/index.html
./events/stancon2018Helsinki/stancon-accommodation.html
./events/stancon2018Helsinki/stancon-code_of_conduct.html
./events/stancon2018Helsinki/stancon-registration.html
./events/stancon2018Helsinki/stancon-submissions.html
./events/stancon2018Helsinki/stancon-travel.html
./events/stancon2019Cambridge/abstracts.html
./events/stancon2019Cambridge/index.html
./events/stancon2020/index.html
./events/stancon2023/index.html
./events/stancon2024/index.html
./events/stancon-code_of_conduct.html
./events/stancon.html
./events/stanconnect_Biomedical/index.html
./events/stanconnect_CogNeuro/index.html
./events/stanconnect_Ecology1/index.html
./events/stanconnect_Ecology2/index.html
./events/stanconnect_SBC/index.html
./events/stancon-submissions.html
./funding/index.html
./getting_started/hello_world.html
./index.html
./interfaces/bayesplot.html
./interfaces/cmdstan.html
./interfaces/index.html
./interfaces/julia-stan.html
./interfaces/loo.html
./interfaces/mathematica-stan.html
./interfaces/math.html
./interfaces/matlab-stan.html
./interfaces/pystan.html
./interfaces/rstanarm.html
./interfaces/rstan.html
./interfaces/rstantools.html
./interfaces/shinystan.html
./interfaces/stan.html
./interfaces/stata-stan.html
./issues/index.html
./misc/warnings.html
./search/index.html
./search/search.html
./shop/index.html
./support/index.html
./tags/index.html
./team/index.html
./users/citations/index.html
./users/documentation/case-studies/bball-hmm.html
./users/documentation/case-studies/boarding_school_case_study.html
./users/documentation/case-studies/cace_one-sided.html
./users/documentation/case-studies/convert_odes.html
./users/documentation/case-studies/curse-dims.html
./users/documentation/case-studies/curse-dims-python.html
./users/documentation/case-studies/dina_independent.html
./users/documentation/case-studies/divergences_and_bias.html
./users/documentation/case-studies/dorazio-royle-occupancy.html
./users/documentation/case-studies/dyadic_irt_model.html
./users/documentation/case-studies/estimating-dynamic-latent-opinion.html
./users/documentation/case-studies/gaussian-process.html
./users/documentation/case-studies/golf.html
./users/documentation/case-studies/gpareto_functions.html
./users/documentation/case-studies/hierarchical_2pl.html
./users/documentation/case-studies/hmm-example.html
./users/documentation/case-studies/HoloML_in_Stan.html
./users/documentation/case-studies/icar_stan.html
./users/documentation/case-studies/identifying_mixture_models.html
./users/documentation/case-studies/jupyter_colab_notebooks_2020.html
./users/documentation/case-studies/Latent_class_case_study.html
./users/documentation/case-studies/losscurves_casestudy.html
./users/documentation/case-studies/lotka-volterra-predator-prey.html
./users/documentation/case-studies/mbjoseph-CARStan.html
./users/documentation/case-studies/mle-params.html
./users/documentation/case-studies/model-based_causal_inference_for_RCT.html
./users/documentation/case-studies/nngp.html
./users/documentation/case-studies/pcm_and_gpcm.html
./users/documentation/case-studies/planetary_motion/planetary_motion.html
./users/documentation/case-studies/plantInteractions.html
./users/documentation/case-studies/pool-binary-trials.html
./users/documentation/case-studies/pool-binary-trials-rstanarm.html
./users/documentation/case-studies/pystan_workflow.html
./users/documentation/case-studies/qr_regression.html
./users/documentation/case-studies/radon_cmdstanpy_plotnine.html
./users/documentation/case-studies/rasch_and_2pl.html
./users/documentation/case-studies/reduce_sum_tutorial.html
./users/documentation/case-studies/rsm_and_grsm.html
./users/documentation/case-studies/rstan_workflow.html
./users/documentation/case-studies/sem.html
./users/documentation/case-studies/soil-knit.html
./users/documentation/case-studies/splines_in_stan.html
./users/documentation/case-studies/tutorial_rstanarm.html
./users/documentation/case-studies/tutorial_twopl.html
./users/documentation/case-studies/weakly_informative_shapes.html
./users/documentation/case-studies.html
./users/documentation/external.html
./users/documentation/index.html
./users/documentation/tutorials.html
./users/index.html
./users/interfaces/bayesplot.html
./users/interfaces/brms.html
./users/interfaces/cmdstan.html
./users/interfaces/index.html
./users/interfaces/julia-stan.html
./users/interfaces/loo.html
./users/interfaces/mathematica-stan.html
./users/interfaces/math.html
./users/interfaces/matlab-stan.html
./users/interfaces/pystan.html
./users/interfaces/rstanarm.html
./users/interfaces/rstan.html
./users/interfaces/rstantools.html
./users/interfaces/shinystan.html
./users/interfaces/stan.html
./users/interfaces/stata-stan.html
./users/issues/index.html
./workshops/ASA2016/index.html
./workshops/az2017/index.html
./workshops/az2017/secret/index.html
./workshops/birds/index.html
./workshops/cppFunctions2018/stan-cpp.html
./workshops/dec2017/1.1.slides.html
./workshops/dec2017/2.2.slides.html
./workshops/dec2017/3.1.slides.html
./workshops/dec2017/exercises_1_2_curse_of_dim.html
./workshops/dec2017/exercises_1_2_MC_simulation.html
./workshops/dec2017/exercises_2_1_model_checking_shinystan.html
./workshops/dec2017/index.html
./workshops/dec2017/spatial_smoothing_icar.html
./workshops/halifax2017/exercise_1.html
./workshops/halifax2017/exercise_2.html
./workshops/halifax2017/exercise_3.html
./workshops/halifax2017/index.html
./workshops/icl_epi/index.html
./workshops/IMPS2016/half1.html
./workshops/IMPS2016/index.html
./workshops/IMPS2016/irt_with_stan.html
./workshops/index.html
./workshops/isba2016/index.html
./workshops/JSM2016/index.html
./workshops/JSM2016/third1.html
./workshops/JSM2016/third2.html
./workshops/meetup/index.html
./workshops/ness2016/index.html
./workshops/nips2015.html
./workshops/nyc/index.html
./workshops/pg2017/index.html
./workshops/pg2017/secret/index.html
./workshops/pydataNYC2017/index.html
./workshops/pydataNYC2019/index.html
./workshops/Rmedicine2018/index.html
./workshops/Rmedicine2018/Stan_tutorial.html
./workshops/stancon2018_hierarchical/index.html
./workshops/stancon2018_intro/index.html
./workshops/stancon2018_intro/radon_slides.html
./workshops/stancon2019_hierarchical/index.html
./workshops/ub_tisi/index.html
./workshops/useR2016/half1.html
./workshops/useR2016/half2.html
./workshops/useR2016/index.html
./workshops/vanderbilt2016/index.html
```

**Comment by WardBrian** [comment]
I guess one way to do this, which might actually be the 'easiest' to verify we did it correctly, is to create a redirects.txt from

```
https://mc-stan.org/404.html
https://mc-stan.org/about/funding/index.html
https://mc-stan.org/about/index.html
https://mc-stan.org/about/logo/index.html
https://mc-stan.org/about/sgb/minutes.html
https://mc-stan.org/about/team/index.html
https://mc-stan.org/citations/index.html
https://mc-stan.org/community/index.html
https://mc-stan.org/developers/index.html
https://mc-stan.org/development/index.html
https://mc-stan.org/documentation/case-studies/curse-dims.html
https://mc-stan.org/documentation/case-studies/dina_independent.html
https://mc-stan.org/documentation/case-studies/divergences_and_bias.html
https://mc-stan.org/documentation/case-studies/dorazio-royle-occupancy.html
https://mc-stan.org/documentation/case-studies/hierarchical_2pl.html
https://mc-stan.org/documentation/case-studies/identifying_mixture_models.html
https://mc-stan.org/documentation/case-studies/Latent_class_case_study.html
https://mc-stan.org/documentation/case-studies/mbjoseph-CARStan.html
https://mc-stan.org/documentation/case-studies/mle-params.html
https://mc-stan.org/documentation/case-studies/pcm_and_gpcm.html
https://mc-stan.org/documentation/case-studies/pool-binary-trials.html
https://mc-stan.org/documentation/case-studies/pool-binary-trials-rstanarm.html
https://mc-stan.org/documentation/case-studies/radon.html
https://mc-stan.org/documentation/case-studies/rasch_and_2pl.html
https://mc-stan.org/documentation/case-studies/rsm_and_grsm.html
https://mc-stan.org/documentation/case-studies/sem.html
https://mc-stan.org/documentation/case-studies/soil-knit.html
https://mc-stan.org/documentation/case-studies/tutorial_twopl.html
https://mc-stan.org/documentation/case-studies/weakly_informative_shapes.html
https://mc-stan.org/documentation/index.html
https://mc-stan.org/events/index.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-ahn-haines-zhang-hBayesDM.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-groves-irt.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-lei-sanders-dawson-ad-attribution.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-margossian-gillespie-ode.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-nicenboim-vasishth-retrieval-models.html
https://mc-stan.org/events/stancon2017-notebooks/stancon2017-sanders-lei-mass-shootings.html
https://mc-stan.org/events/stancon2017-presenter-bios.html
https://mc-stan.org/events/stancon2018/index.html
https://mc-stan.org/events/stancon2018/report.html
https://mc-stan.org/events/stancon2018/stancon-code_of_conduct.html
https://mc-stan.org/events/stancon2018/stancon-submissions.html
https://mc-stan.org/events/stancon2018Helsinki/GpuOptimizedMath.html
https://mc-stan.org/events/stancon2018Helsinki/index.html
https://mc-stan.org/events/stancon2018Helsinki/stancon-accommodation.html
https://mc-stan.org/events/stancon2018Helsinki/stancon-code_of_conduct.html
https://mc-stan.org/events/stancon2018Helsinki/stancon-registration.html
https://mc-stan.org/events/stancon2018Helsinki/stancon-submissions.html
https://mc-stan.org/events/stancon2018Helsinki/stancon-travel.html
https://mc-stan.org/events/stancon2019Cambridge/abstracts.html
https://mc-stan.org/events/stancon2019Cambridge/index.html
https://mc-stan.org/events/stancon2020/index.html
https://mc-stan.org/events/stancon2023/index.html
https://mc-stan.org/events/stancon2024/index.html
https://mc-stan.org/events/stancon-code_of_conduct.html
https://mc-stan.org/events/stancon.html
https://mc-stan.org/events/stanconnect_Biomedical/index.html
https://mc-stan.org/events/stanconnect_CogNeuro/index.html
https://mc-stan.org/events/stanconnect_Ecology1/index.html
https://mc-stan.org/events/stanconnect_Ecology2/index.html
https://mc-stan.org/events/stanconnect_SBC/index.html
https://mc-stan.org/events/stancon-submissions.html
https://mc-stan.org/funding/index.html
https://mc-stan.org/getting_started/hello_world.html
https://mc-stan.org/index.html
https://mc-stan.org/interfaces/bayesplot.html
https://mc-stan.org/interfaces/cmdstan.html
https://mc-stan.org/interfaces/index.html
https://mc-stan.org/interfaces/julia-stan.html
https://mc-stan.org/interfaces/loo.html
https://mc-stan.org/interfaces/mathematica-stan.html
https://mc-stan.org/interfaces/math.html
https://mc-stan.org/interfaces/matlab-stan.html
https://mc-stan.org/interfaces/pystan.html
https://mc-stan.org/interfaces/rstanarm.html
https://mc-stan.org/interfaces/rstan.html
https://mc-stan.org/interfaces/rstantools.html
https://mc-stan.org/interfaces/shinystan.html
https://mc-stan.org/interfaces/stan.html
https://mc-stan.org/interfaces/stata-stan.html
https://mc-stan.org/issues/index.html
https://mc-stan.org/misc/warnings.html
https://mc-stan.org/search/index.html
https://mc-stan.org/search/search.html
https://mc-stan.org/shop/index.html
https://mc-stan.org/support/index.html
https://mc-stan.org/tags/index.html
https://mc-stan.org/team/index.html
https://mc-stan.org/users/citations/index.html
https://mc-stan.org/users/documentation/case-studies/bball-hmm.html
https://mc-stan.org/users/documentation/case-studies/boarding_school_case_study.html
https://mc-stan.org/users/documentation/case-studies/cace_one-sided.html
https://mc-stan.org/users/documentation/case-studies/convert_odes.html
https://mc-stan.org/users/documentation/case-studies/curse-dims.html
https://mc-stan.org/users/documentation/case-studies/curse-dims-python.html
https://mc-stan.org/users/documentation/case-studies/dina_independent.html
https://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html
https://mc-stan.org/users/documentation/case-studies/dorazio-royle-occupancy.html
https://mc-stan.org/users/documentation/case-studies/dyadic_irt_model.html
https://mc-stan.org/users/documentation/case-studies/estimating-dynamic-latent-opinion.html
https://mc-stan.org/users/documentation/case-studies/gaussian-process.html
https://mc-stan.org/users/documentation/case-studies/golf.html
https://mc-stan.org/users/documentation/case-studies/gpareto_functions.html
https://mc-stan.org/users/documentation/case-studies/hierarchical_2pl.html
https://mc-stan.org/users/documentation/case-studies/hmm-example.html
https://mc-stan.org/users/documentation/case-studies/HoloML_in_Stan.html
https://mc-stan.org/users/documentation/case-studies/icar_stan.html
https://mc-stan.org/users/documentation/case-studies/identifying_mixture_models.html
https://mc-stan.org/users/documentation/case-studies/jupyter_colab_notebooks_2020.html
https://mc-stan.org/users/documentation/case-studies/Latent_class_case_study.html
https://mc-stan.org/users/documentation/case-studies/losscurves_casestudy.html
https://mc-stan.org/users/documentation/case-studies/lotka-volterra-predator-prey.html
https://mc-stan.org/users/documentation/case-studies/mbjoseph-CARStan.html
https://mc-stan.org/users/documentation/case-studies/mle-params.html
https://mc-stan.org/users/documentation/case-studies/model-based_causal_inference_for_RCT.html
https://mc-stan.org/users/documentation/case-studies/nngp.html
https://mc-stan.org/users/documentation/case-studies/pcm_and_gpcm.html
https://mc-stan.org/users/documentation/case-studies/planetary_motion/planetary_motion.html
https://mc-stan.org/users/documentation/case-studies/plantInteractions.html
https://mc-stan.org/users/documentation/case-studies/pool-binary-trials.html
https://mc-stan.org/users/documentation/case-studies/pool-binary-trials-rstanarm.html
https://mc-stan.org/users/documentation/case-studies/pystan_workflow.html
https://mc-stan.org/users/documentation/case-studies/qr_regression.html
https://mc-stan.org/users/documentation/case-studies/radon_cmdstanpy_plotnine.html
https://mc-stan.org/users/documentation/case-studies/rasch_and_2pl.html
https://mc-stan.org/users/documentation/case-studies/reduce_sum_tutorial.html
https://mc-stan.org/users/documentation/case-studies/rsm_and_grsm.html
https://mc-stan.org/users/documentation/case-studies/rstan_workflow.html
https://mc-stan.org/users/documentation/case-studies/sem.html
https://mc-stan.org/users/documentation/case-studies/soil-knit.html
https://mc-stan.org/users/documentation/case-studies/splines_in_stan.html
https://mc-stan.org/users/documentation/case-studies/tutorial_rstanarm.html
https://mc-stan.org/users/documentation/case-studies/tutorial_twopl.html
https://mc-stan.org/users/documentation/case-studies/weakly_informative_shapes.html
https://mc-stan.org/users/documentation/case-studies.html
https://mc-stan.org/users/documentation/external.html
https://mc-stan.org/users/documentation/index.html
https://mc-stan.org/users/documentation/tutorials.html
https://mc-stan.org/users/index.html
https://mc-stan.org/users/interfaces/bayesplot.html
https://mc-stan.org/users/interfaces/brms.html
https://mc-stan.org/users/interfaces/cmdstan.html
https://mc-stan.org/users/interfaces/index.html
https://mc-stan.org/users/interfaces/julia-stan.html
https://mc-stan.org/users/interfaces/loo.html
https://mc-stan.org/users/interfaces/mathematica-stan.html
https://mc-stan.org/users/interfaces/math.html
https://mc-stan.org/users/interfaces/matlab-stan.html
https://mc-stan.org/users/interfaces/pystan.html
https://mc-stan.org/users/interfaces/rstanarm.html
https://mc-stan.org/users/interfaces/rstan.html
https://mc-stan.org/users/interfaces/rstantools.html
https://mc-stan.org/users/interfaces/shinystan.html
https://mc-stan.org/users/interfaces/stan.html
https://mc-stan.org/users/interfaces/stata-stan.html
https://mc-stan.org/users/issues/index.html
https://mc-stan.org/workshops/ASA2016/index.html
https://mc-stan.org/workshops/az2017/index.html
https://mc-stan.org/workshops/az2017/secret/index.html
https://mc-stan.org/workshops/birds/index.html
https://mc-stan.org/workshops/cppFunctions2018/stan-cpp.html
https://mc-stan.org/workshops/dec2017/1.1.slides.html
https://mc-stan.org/workshops/dec2017/2.2.slides.html
https://mc-stan.org/workshops/dec2017/3.1.slides.html
https://mc-stan.org/workshops/dec2017/exercises_1_2_curse_of_dim.html
https://mc-stan.org/workshops/dec2017/exercises_1_2_MC_simulation.html
https://mc-stan.org/workshops/dec2017/exercises_2_1_model_checking_shinystan.html
https://mc-stan.org/workshops/dec2017/index.html
https://mc-stan.org/workshops/dec2017/spatial_smoothing_icar.html
https://mc-stan.org/workshops/halifax2017/exercise_1.html
https://mc-stan.org/workshops/halifax2017/exercise_2.html
https://mc-stan.org/workshops/halifax2017/exercise_3.html
https://mc-stan.org/workshops/halifax2017/index.html
https://mc-stan.org/workshops/icl_epi/index.html
https://mc-stan.org/workshops/IMPS2016/half1.html
https://mc-stan.org/workshops/IMPS2016/index.html
https://mc-stan.org/workshops/IMPS2016/irt_with_stan.html
https://mc-stan.org/workshops/index.html
https://mc-stan.org/workshops/isba2016/index.html
https://mc-stan.org/workshops/JSM2016/index.html
https://mc-stan.org/workshops/JSM2016/third1.html
https://mc-stan.org/workshops/JSM2016/third2.html
https://mc-stan.org/workshops/meetup/index.html
https://mc-stan.org/workshops/ness2016/index.html
https://mc-stan.org/workshops/nips2015.html
https://mc-stan.org/workshops/nyc/index.html
https://mc-stan.org/workshops/pg2017/index.html
https://mc-stan.org/workshops/pg2017/secret/index.html
https://mc-stan.org/workshops/pydataNYC2017/index.html
https://mc-stan.org/workshops/pydataNYC2019/index.html
https://mc-stan.org/workshops/Rmedicine2018/index.html
https://mc-stan.org/workshops/Rmedicine2018/Stan_tutorial.html
https://mc-stan.org/workshops/stancon2018_hierarchical/index.html
https://mc-stan.org/workshops/stancon2018_intro/index.html
https://mc-stan.org/workshops/stancon2018_intro/radon_slides.html
https://mc-stan.org/workshops/stancon2019_hierarchical/index.html
https://mc-stan.org/workshops/ub_tisi/index.html
https://mc-stan.org/workshops/useR2016/half1.html
https://mc-stan.org/workshops/useR2016/half2.html
https://mc-stan.org/workshops/useR2016/index.html
https://mc-stan.org/workshops/vanderbilt2016/index.html
```

This should be a list of every single existing page on the mc-stan.org website. Obviously, if a url still exists, we remove it.

Sorry I didn't think of this earlier before suggesting other changes!

**Comment by mitzimorris** [comment]
The list I started from was similar, but restricted to just the markdown files in order to identify content that needed to be copied over:
```
> find . -name "*.md" -print > urls.txt
```

- the `workshops` are out of date.  do we want to archive them somewhere?

- the html files for the case studies have been copied into the new website  - we can add redirects for the individual case studies via the list in `redirects.txt`.

otherwise, I think that this is complete - if someone somewhere has linked to an obscure part of the mc-stan.org website,  they'll get the 404 page and they can search for whatever it was they linked to.

**Comment by WardBrian** [comment]
I think making sure the case studies redirect is important -- they definitely have links floating around on the forums and other places.

The workshops I think it's fine to not preserve. We can either hit a 404 or we could redirect them all to a "this content is no longer available" page? Just so the person looking for it doesn't waste time searching for it's new location when it doesn't have one

**Comment by WardBrian** [comment]
I just pushed an addition of a page that basically says "Sorry, we deleted that" and redirected all the workshop links there, in case someone does end up navigating to one some day.

I found two more sub-urls that I think might be important:
https://mc-stan.org/users/issues/index.html
https://mc-stan.org/misc/warnings.html

In particular, the second of these is linked to by some warning messages in R packages, I think. We might just want to port it over unchanged.

**Comment by mitzimorris** [comment]
> I found two more sub-urls that I think might be important:
> https://mc-stan.org/users/issues/index.html
> https://mc-stan.org/misc/warnings.html

I integrated the content from issues/index.md into the about page - sections "How to report bugs" and "How to contribute".

I agree that the R warnings page contains good stuff.  Git blame shows that this was mostly written by @jgabry.   I think that it would be great if he could review it and update as needed.  If there were a way to generalize this for Python users, that would be awesome, in which case, we could add a section "Best Practices" or "Troubleshooting" as part of the "Learning Resources" section.

File: learn-stan/outdated.qmd [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+You've been directed here because you tried to navigate to a url for a
+learning resource that no longer exists. As Stan evolves, some resources
+are removed from the website.
+
+You can look for more recent replacements under the "Learning Resources" drop down
+above, or in the [documentation](https://mc-stan.org/docs/).
+
+If you **really** need to find the old resource, you can try looking in the
+[GitHub repository history for this site](https://github.com/stan-dev/stan-dev.github.io/commits/master/).
```
Comment (mitzimorris) [inline learn-stan/outdated.qmd:62]: Nice!   many thanks!

**Comment by jgabry** [comment]
Yeah the page https://mc-stan.org/misc/warnings.html page was written by me, Aki, Andrew, Ben, Martin, and various other developers and with feedback from users (I may have committed most of it to GitHub, but many people contributed to writing it and commenting on it). It's already not intended to be R specific (it intentionally doesn't contain any code). I think basically all the recommendations already apply to Python too (and it already mentions arViz and ArviZ.jl when it mentions bayesplot) . It's mostly about how to interpret diagnostics and update Stan programs, so should be useful to anyone using Stan. For a long time CmdStanR and RStan have been pointing to that page in the warning message if there are divergences/max treedepths, etc, so it would be a bit of a mess if that URL were to break. (If the URL is going to change we would at least need to update the messages to a new URL before the website goes live.)

**Comment by mitzimorris** [comment]
thanks @jgabry  - I will review and edit this page and add it to the "Learning Resources" section; quarto provides hooks so that it will create the redirects for us.

**Comment by jgabry** [comment]
Great, thank you @mitzimorris!

**Review by WardBrian — APPROVED** [review]

**Pull request merged**